hazelcast/src/main/java/com/hazelcast/config/alto/AltoSocketConfig.java

```diff
@@ -1,58 +1,86 @@
 package com.hazelcast.config.alto;
 
-import com.hazelcast.config.InvalidConfigurationException;
+import com.hazelcast.spi.annotation.Beta;
 
 import javax.annotation.Nonnull;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
+import static com.hazelcast.internal.util.Preconditions.checkNotNull;
+import static com.hazelcast.internal.util.Preconditions.checkPositive;
+
+/**
+ * Socket configuration for Alto. In Alto, each eventloop has its own
+ * socket.
+ *
+ * @see AltoConfig
+ * @since 5.3
+ */
+@Beta
 public class AltoSocketConfig {
-    private static final String DEFAULT_PORT_RANGE = "11000-21000";
-    private static final int DEFAULT_RECEIVE_BUFFER_SIZE = 128 * 1024;
-    private static final int DEFAULT_SEND_BUFFER_SIZE = 128 * 1024;
-
-    private String portRange = DEFAULT_PORT_RANGE;
-    private int receiveBufferSize = DEFAULT_RECEIVE_BUFFER_SIZE;
-    private int sendBufferSize = DEFAULT_SEND_BUFFER_SIZE;
-
+    private String portRange = "11000-21000";
+    private int receiveBufferSizeKB = 128;
+    private int sendBufferSizeKB = 128;
+
+    /**
+     * This method gets the possible port range for alto sockets to bind.
+     *
+     * @return the port range string
+     */
     public String getPortRange() {
         return portRange;
     }
 
+    /**
+     * This method sets the possible port range for alto sockets to bind.
```

Comment:
Don't add 'this method'. Just write 'Sets the allowed range for alto server sockets'.